subsurface-helper.cpp

```diff
@@ -111,6 +111,8 @@ void run_ui()
 	ctxt->setContextProperty("connectionListModel", &connectionListModel);
 	ctxt->setContextProperty("logModel", MessageHandlerModel::self());
 
+	qmlRegisterUncreatableType<QMLManager>("org.subsurfacedivelog.mobile",1,0,"ExportType","Enum is not a type");
```

Comment:
the order of commits means that the previous few commits won't actually work.
Why isn't this part of the commit that adds that enum?

Reply:
What does not work ?
The commits are in this order:
1) add enum to qmlmanager.h (222ab86cc mobile-widgets: add export functionality to qmlmanager)
2) make enum available to qml (a66adbbb5 startup mobile: make ExportType enum available to QML)
3) use it in QML (bd091859c mobile-widgets/qml: add export page)

It is not part of the commit that adds the enum, because having qmlmanager.h and subsurface-helper.cpp in the same commit seems wrong to me (from a bisect pow).

Reply:
I must have confused myself reading the code. Sorry